modules/log-collector-http-server.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/cluster-logging-collector.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log-collector-http-server_{context}"]
+= Configuring the collector to listen for connections as an HTTP server
```

Comment:
Is there any other use case for the collector to be listening as an HTTP server other than to collect audit logs? If not, should "audit logs" be incorporated into this title, if that's what users might be looking for?

"Configuring the collector to receive audit logs as an HTTP server" or something?

Reply:
Good call. Originally it was more general, but eng changed it because only audit logs are possible :) updated